packages/file-type-icons/src/FileTypeIconMap.ts

```diff
@@ -404,8 +405,10 @@ export const FileTypeIconMap: { [key: string]: { extensions?: string[] } } = {
       'ithmb',
       'jar',
       'kb',
+      'layer',
```

Comment:
I don't think this is a sysfile. Carlos mentioned CAD but I can't really find any references to it.